diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -226,6 +238,36 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule("newlines", conf
         }
     }
 
+    /**
+     * Check that newline is not placed before or after a colon
+     */
+    private fun handleColon(node: ASTNode) {
+        val prevNewLine = node
+            .parent({ it.treePrev != null }, strict = false)
+            ?.treePrev
+            ?.takeIf {
+                it.elementType == WHITE_SPACE && it.text.contains("\n")
+            }
+        prevNewLine?.let { colon ->
+            WRONG_NEWLINES.warnAndFix(configRules, emitWarn, isFixMode, "newline shouldn't be placed before colon", node.startOffset, node) {
+                colon.treeParent.removeChild(colon)
+
+                colon.treeParent?.let {
+                    val nextNewLine = node
+                        .parent({ it.treeNext != null }, strict = false)
+                        ?.treeNext
+                        ?.takeIf {
+                            it.elementType == WHITE_SPACE && it.text.contains("\n")
+                        }
+                    nextNewLine?.let {
```

Comment:
Please add some comment to describe why we are removing next newline here, wile not checking it specifically